blueflood-http/src/test/resources/sample_enums_payload.json

```diff
@@ -1,6 +1,6 @@
 {
     "tenantId":"333333",
-    "timestamp":1439231324000,
+    "timestamp":1389211230,
```

Comment:
This is actually just changing the timestamp from "10 Aug 2015 18:28:44 GMT" to "08 Jan 2014 20:00:30 GMT", to match the other json samples in the blueflood-http/src/test/resources/ directory, so that we can reuse the samples in other tests without having to extend the date range to a huge number.